lametro/views.py

```diff
@@ -100,9 +106,16 @@ def extra_context(self):
             "start_date"
         )
         extra["form"] = LAMetroCouncilmaticSearchForm()
+        extra["alert_form"] = self.get_form()
 
         return extra
 
+    def form_valid(self, form):
+        success_url = self.get_success_url()
+        form.save()
+
+        return HttpResponseRedirect(success_url)
```

Comment:
Apart from saving the form, this is the default behavior, so you can extend (add extra stuff to) the method instead of overriding (completely replacing) it:

```suggestion
        form.save()
        return super().form_valid()
```

Reply:
Ohh gotcha, I'll make sure I make more of a conscious decision to extend or override these kinds of methods!